lib/compilers/babel-compiler.js

```diff
@@ -23,7 +23,16 @@ module.exports = function compileBabel (scriptContent, inputSourceMap, inlineCon
     babelOptions = Object.assign(babelOptions, babelOverWrites)
   }
 
-  const res = babel.transform(scriptContent, babelOptions)
```

Comment:
Can you revert to the previous code? babel.transform will use babel 6 transform if babel-core is babel 6, and babel 7 if babel-core is @bridge.

Reply:
My tests were failing when i reveretd back :/
![image](https://user-images.githubusercontent.com/13895272/50542017-cea12280-0bba-11e9-98fc-0acc0a800f86.png)

Reply:
I reverted back to original code, except this chunk
![image](https://user-images.githubusercontent.com/13895272/50542536-5e010280-0bc8-11e9-9dd1-253b51254230.png)
This to only reproduce the issue in test

Now the snapshot test is failing, can you check the codes for assesment